test/src/responder/gasPriceEstimator.test.ts

```diff
@@ -127,8 +128,7 @@ describe("GasPriceEstimator", () => {
             startBlock: 7
         });
     };
-
-    it("estimate", async () => {
+        fnIt<GasPriceEstimator>(e => e.estimate, "", async () => {
```

Comment:
An extra tab here. Salvatore and I use an vs code plugin called "Prettier", it stops use making different formatting changes from each other. You should add it then set the following options in your vs code settings:

```
"prettier.tabWidth": 4,
    "prettier.printWidth": 120,
    "[typescript]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[json]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
```